src/ts/address-search.ts

```diff
@@ -1,1240 +1,1441 @@
+          for (let i = 3; i < lines.length; i++) {
+            inputData.result.address.address_line_3 += lines[i];
+          }
+        }
+
+        this.result.show(inputData);
+        this.result.updateHeading(this.options.formattedAddressContainer.manualHeadingText);
+      },
+      // Create and return an address line object with the key as the label
+      formatManualAddressLine: function (lines, i) {
+        const key = defaults.addressLineLabels[i];
+        const lineObject = {};
+        lineObject[key] = lines[i] || '';
+        return lineObject;
+      }
+    };
+
+    // Create the picklist list (and container) and inject after the input
+    this.picklist.createList = () => {
+      // If Singleline mode is used, then append the picklist after the last input field, otherwise use the first one
+      const position = this.searchType === AddressValidationMode.SINGLELINE ? this.inputs.length - 1 : 0;
+
+      const container = document.createElement('div');
+      container.classList.add('address-picklist-container');
+      this.picklist.container = container;
+
+      // Insert the picklist container after the input
+      this.inputs[position].parentNode.insertBefore(this.picklist.container, this.inputs[position].nextElementSibling);
+
+      const list = document.createElement('div');
+      list.classList.add('address-picklist');
+      // Append the picklist to the inner wrapper
+      this.picklist.container.appendChild(list);
+
+      // Add a class to the input to denote that a picklist with suggestions is being shown
+      this.inputs[position].classList.add('showing-suggestions');
+
+      list.addEventListener('keydown', this.picklist.checkEnter);
+      return list;
+    };
+
+    // Create a new picklist item/row
+    this.picklist.createListItem = (item: PicklistItem) => {
+      const row = document.createElement('div');
+      row.innerHTML = this.picklist.addMatchingEmphasis(item);
+
+      // Store the Format URL if it exists, otherwise use the global_address_key as a "refinement" property
+      if (item.format) {
+        row.setAttribute('format', item.format);
+      } else if (item.global_address_key) {
+        row.setAttribute('refine', item.global_address_key);
+      }
+      return row;
+    };
+
+    // Create a new picklist item/row for what3words
+    this.picklist.createWhat3WordsListItem = (item: What3WordsPickList) => {
+      const row = document.createElement('div');
+      const name = document.createElement('div');
+      const description = document.createElement('div');
+
+      row.className = this.what3WordsKeyword;
+      name.className = "what3WordsName";
+      description.className = "what3WordsDescription";
+
+      name.innerHTML = "///" + item.what3words.name;
+      description.innerHTML = item.what3words.description;
+
+      row.appendChild(name);
+      row.appendChild(description);
+
+      return row;
+    };
+
+
+    // Create a new picklist item/row for what3words

```

Comment:
I think this comment is copied from the w3w one?

Reply:
yes, changed it